src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/ApprenticeController.cs

```diff
@@ -249,5 +254,38 @@ public IActionResult Sent()
             var model = TempData[nameof(ConfirmViewModel.NewEmployerName)] as string;
             return View(nameof(Sent), model);
         }
+
+        [HttpGet]
+        [Route("{CohortReference}/{apprenticeshipHashedId}/Delete", Name = RouteNames.ApprenticeDelete)]        
```

Comment:
adding cohorReference to the route here doesn't really make sense and is inconsistent

Reply:
this action should be move to DraftApprenticeshipController

Reply:
moved to DraftApprenticeshipController